src/context/fetch.ts

```diff
@@ -26,14 +28,24 @@ const createFetchRequestParameters = (input: MockedRequest): RequestInit => {
     return requestParameters
   }
 
-  if (
-    typeof body === 'object' ||
-    typeof body === 'number' ||
-    typeof body === 'boolean'
-  ) {
-    requestParameters.body = JSON.stringify(body)
-  } else {
-    requestParameters.body = body
+  switch (typeof body) {
+    case 'object':
+      if (input.headers?.get('content-type')?.includes('multipart/form-data')) {
+        const formData = new FormData()
```

Comment:
So the issue was that `req.body` was a string? Because all intercepted requests have a string body. 

And here you're reconstructing a `FormData` instance to assign to the proxied request's body without even touching the `content-type` header. Do I understand this correctly?

Reply:
Correct